src/ansiblelint/cli.py

```diff
@@ -127,6 +127,10 @@ def get_cli_parser() -> argparse.ArgumentParser:
                         default=False,
                         action='store_true',
                         help="parseable output in the format of pep8")
+    parser.add_argument('--parseable-codeclimate', dest='parseable_codeclimate',
+                        default=False,
+                        action='store_true',
+                        help="parseable output in the format of codeclimate json report")
```

Comment:
Parseable severity was a historical mistake and I do not want to clutter the CLI options with very specific cases. 

Instead you could add a new formatter and extend `-f {rich,plain,rst}` to add codeclimate to the list of allowed options, without adding new command line options.